superset/assets/src/explore/components/controls/DatasourceControl.jsx

```diff
@@ -211,6 +211,18 @@ export default class DatasourceControl extends React.PureComponent {
             />
           </a>
         </OverlayTrigger>
+        {this.props.datasource.type === 'table' && <OverlayTrigger
```

Comment:
nit (optional): personally I like  to break before an incomplete tag as in `\n<OverlayTrigger` but will defer on the linter here